frontend/src/components/form/base/EColorPicker.vue

```diff
@@ -11,22 +11,18 @@ Displays a field as a color picker (can be used with v-model)
     :parse-picker="parsePicker"
     :vee-rules="veeRules || { regex: /#([a-f0-9]{3}){1,2}\b/i }"
     :required="required"
+    button-aria-label-i18n-key="components.form.base.eColorPicker.openPicker"
     @input="$emit('input', $event)"
   >
-    <template slot-scope="picker">
+    <template #default="picker">
       <v-card>
         <v-color-picker
-          v-if="picker.showPicker"
-          :value="picker.value"
+          :value="removeAlpha(picker.value || '')"
           flat
-          @input="picker.on.input"
+          @input="picker.onInput"
         />
-        <v-spacer />
-        <v-btn text color="primary" data-testid="action-cancel" @click="picker.on.close">
-          {{ $tc('global.button.cancel') }}
-        </v-btn>
-        <v-btn text color="primary" data-testid="action-ok" @click="picker.on.save">
-          {{ $tc('global.button.ok') }}
+        <v-btn text color="primary" @click="picker.close">
```

Comment:
If we still need a close button, why not keeping the ok/cancel buttons, so the user has both functionalities? I don't think this is necessarily an improvement. But I can understand, there are different views on this.

Reply:
The pickers as I understand them intuitively (and find twice as efficient to use, see the screencaps in the PR description) is that they are not modal dialogs, but rather input controls, just like a text field or number input. So there is no "old saved value" to go back to with a cancel button. When interacting with the picker, it immediately updates the text field. Interacting with the picker is in every way interchangeable with just editing the value in the text field. This leads to better keyboard accessibility and testability.

This difference, that the pickers are modal dialogs instead of "specialized on-screen keyboards" for entering values into the picker text field, is the thing I find the most annoying by far with the current picker implementation, and it's a big part of the reason why I wanted to make these improvements in the first place.

Reply:
I think, once we have a range datetime picker, we can reintroduce the cancel and ok behavior. If needed.

I agree with @carlobeltrame. As I wrote in my comment, the current PR has some minor visual problems. E.g. I'd rather have a floating close "X" outside the panel instead of this "Cancel" button.